classes/external/end_meeting.php

```diff
@@ -69,7 +69,7 @@ public static function execute_parameters(): external_function_parameters {
      * @throws \restricted_context_exception
      */
     public static function execute(
-        int $bigbluebuttonbnid,
+        int $bigbluebuttonbnid, // TODO: we might want to remove the $bigbluebuttonid here as we can just rely on meetingid.
```

Comment:
We don't want to expose the actual values. This is why we should be using the Moodle ids

Reply:
Yes but we don't actually use the bigbluebuttonid. Either that or we need to add a new groupid parameter. The meetingid is providing both. As far as I know the callback do not use the bbbid.

Reply:
You are right. For all the actions related to meetings we can rely on meetingid (for now) as it is built on the fly and then forgotten after the meeting ends.